AndorsTrail/src/com/gpl/rpg/AndorsTrail/model/item/ItemType.java

```diff
@@ -6,7 +6,7 @@
 
 public final class ItemType {
 
-	public static enum DisplayType {
```

Comment:
There's just no need to do that, since inner enums are static anyway. http://stackoverflow.com/questions/663834/in-java-are-enum-types-inside-a-class-static
But I guess if that's a common code style in AT, I could revert that part.